### ISSUE TITLE
[BUG] qBittorent API not working

### ISSUE THREAD

**Issue by GreysonMannix** [rogerfar/rdt-client#453]
**What version are you using?**

2.0.6.5

**Wat OS are you running?**

Debian 11 Bullseye

**Are you using Docker or as a service?**

Docker image tag "latest"

**Which debrid provider are you using?**

Alldebrid

**Which downloader are you using?**

*arr

**Please attach a log file here with the log setting set to debug**

2024-04-08 15:09:00.6|Warn|RadarrErrorPipeline|Invalid request Validation failed: 
 -- : Failed to get the list of torrents: Failed to connect to qBittorrent, check your settings.
2024-04-08 15:09:00.8|Warn|HttpClient|HTTP Error - Res: HTTP/1.1 [GET] http://192.168.178.2:6500/api/v2/torrents/info?category=radarr: 500.InternalServerError (53 bytes)
Value was either too large or too small for an Int64.
2024-04-08 15:09:00.8|Error|QBittorrent|Failed to get torrents

[v5.3.6.8612] NzbDrone.Core.Download.Clients.DownloadClientException: Failed to connect to qBittorrent, check your settings.
 ---> NzbDrone.Common.Http.HttpException: HTTP request failed: [500:InternalServerError] [GET] at [http://192.168.178.2:6500/api/v2/torrents/info?category=radarr]
   at NzbDrone.Common.Http.HttpClient.ExecuteAsync(HttpRequest request) in ./Radarr.Common/Http/HttpClient.cs:line 119
   at NzbDrone.Common.Http.HttpClient.Execute(HttpRequest request) in ./Radarr.Common/Http/HttpClient.cs:line 128
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.ProcessRequest(HttpRequestBuilder requestBuilder, QBittorrentSettings settings) in ./Radarr.Core/Download/Clients/QBittorrent/QBittorrentProxyV2.cs:line 379
Value was either too large or too small for an Int64.
   --- End of inner exception stack trace ---
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.ProcessRequest(HttpRequestBuilder requestBuilder, QBittorrentSettings settings) in ./Radarr.Core/Download/Clients/QBittorrent/QBittorrentProxyV2.cs:line 394
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.ProcessRequest[TResult](HttpRequestBuilder requestBuilder, QBittorrentSettings settings) in ./Radarr.Core/Download/Clients/QBittorrent/QBittorrentProxyV2.cs:line 363
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.GetTorrents(QBittorrentSettings settings) in ./Radarr.Core/Download/Clients/QBittorrent/QBittorrentProxyV2.cs:line 104
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.TestGetTorrents() in ./Radarr.Core/Download/Clients/QBittorrent/QBittorrent.cs:line 585

Confirmed also requesting the url from the browser.
Fixed by reverting back to 2.0.6.3

**Comment by rogerfar** [comment]
Do you mind sharing the magnet that is causing this through pastebin? I tried reproducing it but couldn't.

**Comment by GreysonMannix** [comment]
It's not a magnet, the API doesn't work at all for the arr apps.

First thing the arr app is doing is retrieving from the client the list of active torrents for the category:

`http://192.168.178.2:6500/api/v2/torrents/info?category=radarr`

This fails with HTTP error 500 and answering in the body:

`Value was either too large or too small for an Int64.`

I tried without category as well from the browser with the same result.
Works fine on 2.0.6.3.

Maybe without categories defined it works?

**Comment by GreysonMannix** [comment]
Well yes, I have some torrents in download, I didn't remove them.
Maybe it's failing when reading them.
Let me try again with the current queue.

**Comment by GreysonMannix** [comment]
Yes and now it works.
And I have an idea what could have been the issue...
Probably a bad french torrent site, `torrent9.to`.
I disabled it yesterday when I noticed it was fake but today there were probably still a couple of failed downloads in the queue.
If you pick any magnet from there it will probably cause the issue (they are all fake seems).
What is weird is that it's causing 2.0.6.5 to fail and not 2.0.6.3.
